src/GovUk.Education.ExploreEducationStatistics.Content.Api.Tests/Controllers/DataSetFilesControllerTests.cs

```diff
@@ -1641,12 +1652,15 @@ public async Task DataSetFileMetaCorrectlyReturned_Success()
                 dataSetFileMetaViewModel.GeographicLevels
                     .AssertDeepEqualTo(originalMeta!.GeographicLevels);
 
-                dataSetFileMetaViewModel.TimePeriod.AssertDeepEqualTo(
-                    new DataSetFileTimePeriodViewModel
+                dataSetFileMetaViewModel.TimePeriodRange.AssertDeepEqualTo(
+                    new DataSetFileTimePeriodRangeViewModel
                     {
-                        TimeIdentifier = originalMeta.TimeIdentifier.GetEnumLabel(),
-                        From = TimePeriodLabelFormatter.Format(originalMeta.Years.First(), originalMeta.TimeIdentifier),
-                        To = TimePeriodLabelFormatter.Format(originalMeta.Years.Last(), originalMeta.TimeIdentifier),
+                        From = TimePeriodLabelFormatter.Format(
+                            originalMeta.TimePeriodRange.Start.Period,
+                            originalMeta.TimePeriodRange.Start.TimeIdentifier),
+                        To = TimePeriodLabelFormatter.Format(
+                            originalMeta.TimePeriodRange.End.Period,
+                            originalMeta.TimePeriodRange.End.TimeIdentifier),
                     });
 
                 dataSetFileMetaViewModel.Filters
```

Comment:
Similar to above - can just use `Assert.Equal`.

Same for indicators below as well!